src/site/xdoc/release_notes.xml

```diff
@@ -21,6 +21,7 @@
        <ul>
           <li>WatchedCDXSource now monitors directories for new CDX files. <a href="https://github.com/iipc/openwayback/pull/181">#181</a></li>
           <li>Minor fixes to replace hardcoded port numbers aun URL prefixes with placeholders. <a href="https://github.com/iipc/openwayback/pull/223">#223</a></li>
+          <li>MAVEN compile and build now requires Java 7.<a href="https://github.com/iipc/openwayback/pull/181">#178</a></li>
```

Comment:
Cut´n paste error…

Fixed.

Fra: Kristinn Sigurðsson [mailto:notifications@github.com]
Sendt: 15. april 2015 14:28
Til: iipc/openwayback
Kopi: Roger Mathisen
Emne: Re: [openwayback] Issue178 require java7 (#234)


In src/site/xdoc/release_notes.xml<https://github.com/iipc/openwayback/pull/234#discussion_r28414585>:

> @@ -21,6 +21,7 @@

>         <ul>

>            <li>WatchedCDXSource now monitors directories for new CDX files. <a href="https://github.com/iipc/openwayback/pull/181">#181</a></li>

>            <li>Minor fixes to replace hardcoded port numbers aun URL prefixes with placeholders. <a href="https://github.com/iipc/openwayback/pull/223">#223</a></li>

> +          <li>MAVEN compile and build now requires Java 7.<a href="https://github.com/iipc/openwayback/pull/181">#178</a></li>

This links to the wrong issue. And the text should really just be "Require Java 7" This also means that Java 7 is needed to run OWB.

—
Reply to this email directly or view it on GitHub<https://github.com/iipc/openwayback/pull/234/files#r28414585>.